src/Login/containers/LoginContainer.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Login from '../components/LoginComponent';
+import { loginUtil } from '../redux/validations';
+import loginAction from '../redux/actions/LoginAction';
+
+export class LoginContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errors: {
+        invalid_email:false,
+      }
+    };
+  }
+  onChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+
+    const target = event.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
+    const name = target.name;
+    const { errors } = this.state;
+    switch(name) {
+      case "email":
+        if (!this.checkValidEmail(value)) {
+          this.setState({ errors: { userEmail: 'Please provide a valid email address' } });
+          errors.invalid_email = true;
+        }else{
+          this.setState({ errors: { userEmail: '' } });
+          errors.invalid_email = false;
+        }
+        break;
+      default:
+        break;
+    }
+
+    this.setState({ [event.target.name]: event.target.value });
+
+  }
+
+  checkLoginStatus = () => {
+    const {isloggedIn} = this.props.login;
+    if(!isloggedIn){
+      this.setState({ is_form_submited: false});
```

Comment:
What happens if the form is submitted but I was not logged in?

Reply:
This method has been removed.
 This is because before l was not using my props in the right way but currently l have corrected them and basing on `islogged` being true if your logged or false if  not `islogged`.